ajax/add-ftp-user.php

```diff
@@ -0,0 +1,37 @@
+<?php
+require('../template/top.php');
+require('../api/groupme-funcs.php');
+
+if (isset($_POST)) {
+	$username = @$_POST['username'];
+	$password = @$_POST['password'];
+
+	do {
+		$response = json_decode(file_get_contents('https://www.google.com/recaptcha/api/siteverify?secret=' . GOOGLE_RECAPTCHA_KEY . '&response='.$captcha."&remoteip=".$_SERVER['REMOTE_ADDR']), true);
```

Comment:
This captcha isn't used (take a look at the dev-web-logs)

Reply:
taken out now